mbq/client/contrib/permissions.py

```diff
@@ -33,6 +33,12 @@ class RefSpec:
     type: Optional[RefType] = None
 
 
+@dataclass
+class StaffPermissionsDoc:
+    is_superuser: bool
```

Comment:
We should talk about if we still want this flag, or just rely on the existence of a `SUPER_USER` scope in the permissions list.

Reply:
talked IRL, gonna use the flag which is powered by a super user scope in os-core